CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#149] Clarified escaping rules for running the operator locally
   (see [here](docs/development/development_guide_en.md) or [here](.env.template))
 
+### Fixed
+- [#152] Fix a bug where the health routine marked a dogu as available if the deployment was scaled to 0.
```

Comment:
Maybe `- [#152] The health routine no longer marks a dogu as available if the deployment was scaled to 0.`